Sources/Clappr/Classes/Base/SharedData.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+@objc open class SharedData: NSObject {
+    @objc public var storeDictionary = [String: Any]()
+    @objc public weak var container: Container?
```

Comment:
Hey @dezouzart, why `SharedData` has a weak reference to container?

Reply:
Hey @dezouzart , I'm going to aprove this PR. In the future, we can think in another approach to not need this weak reference.